docs/user-guide/reminders-and-external-events.rst

```diff
@@ -0,0 +1,257 @@
+:desc: Learn how to use external events and schedule reminders.
+
+.. _reminders-and-external-events:
+
+Reminders and external events
+=============================
+
+.. edit-link::
+
+In this guide, you will learn how to let your assistant respond to external events (other applications, sensors, etc.) or to remind you about things after a given period of time.
+
+.. contents::
+   :local:
+
+
+.. _external-event-guide:
+
+External events
+---------------
+
+Let's say you want to send a message from some other device to change the course of an ongoing conversation.
+For example, some moisture-sensor attached to a Raspberry Pi should inform your personal assistant that your favourite
+plant needs watering, and your assistant should then relay this message to you.
+
+To do this, your Raspberry Pi needs to send a message to the `trigger_intent endpoint <../../api/http-api/#operation/triggerConversationIntent>`_ of your conversation.
+As the name says, this injects a user intent (possibly with entities) into your conversation.
+So for Rasa it is almost as if you had entered a message that got classified with this intent and these entities.
+Rasa then needs to respond to this input with an action such as ``action_warn_dry``.
+The easiest and most reliable way to connect this action with the intent is via the :ref:`mapping-policy`.
+
+
+.. _getting-conversation-id:
+
+Getting the conversation ID
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first thing we need is the Session ID of the conversation that your sensor should send a notification to.
+An easy way to get this is to define a custom action (see :ref:`custom-actions`) that displays the ID in the conversation.
```

Comment:
This seems hacky. Why would you need the session id? Can't a user say "please keep me updated when a plant gets dry" and it's automatically scheduled for the right conversation id?

Reply:
@wochinge if you are just using regular reminders (second section), you don't need to get the ID. This is an example for External events, where you need the convo id. But I think we kindof have the same question: https://github.com/RasaHQ/rasa/pull/5189/files/9b87d13c841149082173cac4a5b975218c8c013d#r378138647